src/python/tools/db2graph/marius_db2graph.py

```diff
@@ -0,0 +1,709 @@
+import argparse
+import pandas as pd
+import mysql.connector
+from mysql.connector import errorcode
+import psycopg2
+from pathlib import Path
+import uuid
+import re
+from omegaconf import OmegaConf
+from pathlib import Path
+import time
+import logging
+import psutil
+import sys
+
+INVALID_ENTRY_LIST = ["0", None, "", 0, "not reported", "None", "none"]
+FETCH_SIZE = 10000
+
+def set_args():
+    parser = argparse.ArgumentParser(
+                description='Db2Graph', prog='db2graph')
```

Comment:
Add a brief description of the tool here and some usage info

Reply:
Added the description and pointed to the documentation also.